### ISSUE TITLE
Module yaml fragment: block mapping parsing error

### ISSUE THREAD

**Issue by priyanka-amzn** [aws-cloudformation/cloudformation-cli#693]
4 files are part of the attached zip file: 2 module fragments(1 yaml, 1 json) and 2 rpdk(1 for yaml case, 1 for json case) log files.

When I ran `cfn-validate` for `ec2withsub.json` fragment, the output was:
```
Module fragment is probably valid, but there are warnings/errors from cfn-lint (https://github.com/aws-cloudformation/cfn-python-lint):
	At least one of [ImageId, LaunchTemplate] should be specified for Resources/EC2Instance/Properties (from rule E2522: Check Properties that need at least one of a list of properties)
	Fn::Sub isn't needed because there are no variables at Resources/EC2Instance/Properties/UserData/Fn::Base64/Fn::Sub (from rule W1020: Sub isn't needed if it doesn't have a variable defined)
```
But when I ran `cfn-validate` for `ec2withsub.yaml` which has the same cfn resource as `ec2withsub.json` but is in yaml, the output was:
```
Invalid template fragment: Fragment file '/Users/priyaha/testDate/fragments/sample.json' is invalid: while parsing a block mapping
  in "<unicode string>", line 8, column 9:
            Fn::Base64: Sub |
            ^
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 10, column 11:
              sudo yum install -y https://s3.r ... 
              ^
```

[module fragments and logs.zip](https://github.com/aws-cloudformation/cloudformation-cli/files/6046847/module.fragments.and.logs.zip)

**Comment by MalikAtalla-AWS** [comment]
Thanks for reporting this. I'm able to reproduce the issue from your description.

**Comment by MalikAtalla-AWS** [comment]
Will be resolved in versions >= 0.2.6
